weather_mv/loader_pipeline/bq.py

```diff
@@ -151,10 +149,17 @@ def validate_arguments(cls, known_args: argparse.Namespace, pipeline_args: t.Lis
                             region=pipeline_options_dict.get('region'))
             logger.info('Region validation completed successfully.')
 
+        if known_args.zarr:
+            # Encourage use of correct output_path format.
+            _, out_ext = os.path.splitext(known_args.output_path)
```

Comment:
We may need to define a zarr output format for namespace kwargs

Reply:
I don't quite understand. What are namespace kwargs?

Reply:
Hey Alex, I was referring to the error in the build: AttributeError: 'Namespace' object has no attribute 'output_path'. Seems like we may have to define it.